lib/ember-cli/app.rb

```diff
@@ -84,45 +82,34 @@ def wait
       MSG
     end
 
-    def ember_path
-      @ember_path ||= app_path.join("node_modules", ".bin", "ember").tap do |path|
-        fail <<-MSG.strip_heredoc unless path.executable?
-          No local ember executable found. You should run `npm install`
-          inside the #{name} app located at #{app_path}
-        MSG
+    def method_missing(method_name, *)
+      if path_method = supported_path_method(method_name)
+        paths.public_send(path_method)
+      else
+        super
```

Comment:
@rondale-sc ok, delegate is not gonna work because ActiveSupport doesn't support suffixes :(

I've send a PR adding suffix support (https://github.com/rails/rails/pull/19608), but for now we'll have to deal with messy method_missing stuff.